src/get-started/wordcount-example.md

```diff
@@ -300,7 +546,15 @@ The following sections explain these key concepts in detail, and break down the
 
 ### Logging
 
-Each runner may choose to handle logs in its own way.
+Each runner may choose to handle logs in its own way. 
+
+#### Direct Runner
+
+If you execute your pipeline using `DirectRunner`, it will print the log messages directly to your local console.
+
+#### Dataflow Runner
+
+If you execute your pipeline using `DataflowRunner`, you can use Google Cloud Logging. Google Cloud Logging (currently in beta) aggregates the logs from all of your Dataflow job's workers to a single location in the Google Cloud Platform Console. You can use Cloud Logging to search and access the logs from all of the Compute Engine instances that Dataflow has spun up to complete your Dataflow job. You can add logging statements into your pipeline's `DoFn` instances that will appear in Cloud Logging as your pipeline runs.
```

Comment:
Indeed, this is a conscious shift back to Cloud Logging from Stackdriver Logging. is that intentional, @hadarhg ?